huaweicloud/services/ga/resource_huaweicloud_ga_address_group.go

```diff
@@ -330,9 +334,184 @@ func resourceIpAddressGroupUpdate(ctx context.Context, d *schema.ResourceData, m
 		}
 	}
 
+	if d.HasChange("ip_addresses") {
+		oldIpsRaw, newIpsRaw := d.GetChange("ip_addresses")
+		addIpsRaw := newIpsRaw.(*schema.Set).Difference(oldIpsRaw.(*schema.Set))
+		removeIpsRaw := oldIpsRaw.(*schema.Set).Difference(newIpsRaw.(*schema.Set))
+
+		if removeIpsRaw.Len() > 0 {
+			err = removeIps(ctx, d, meta, updateIpAddressGroupClient, removeIpsRaw.List())
+			if err != nil {
+				return diag.FromErr(err)
+			}
+		}
+
+		if addIpsRaw.Len() > 0 {
+			err = addIps(ctx, d, meta, updateIpAddressGroupClient, addIpsRaw.List())
```

Comment:
what will be happened if more than 20 CIDR blocks added at a time?

Reply:
done